acceptance-tests/src/test/resources/features/site/search.feature

```diff
@@ -1,3 +1,4 @@
+@WIP
```

Comment:
Is it ok to merge tests in with the WIP annotation?

Reply:
Removed that now.